channeldb/reports.go

```diff
@@ -74,6 +77,23 @@ const (
 	// claimed, for example and anchor where anybody can spend, was claimed
 	// by another party.
 	ResolverOutcomeLost
+
+	// ResolverOutcomeFirstStage indicates that we broadcast the first stage
+	// of a two stage htlc claim.
+	ResolverOutcomeFirstStage
+
+	// ResolverOutcomeInvalid indicates that we deemed contract invalid
+	// and did not try to resolve it further. This is the case for htlcs
+	// that we cannot decode.
+	ResolverOutcomeInvalid
+
+	// ResolverOutcomeTimeout indicates that we timed out a contract on
+	// chain.
+	ResolverOutcomeTimeout
```

Comment:
These outcomes aren't expressed from our perspective, but rather exactly what happened on chain, eg incoming + claim = us, outgoing + claim = remote. We could have these from our perspective, but then I think we'd need local claim and remote claim as outcomes? I think that's less clear, since we're dealing with the specific on chain actions here.